apps/qubit/modules/informationobject/templates/multiFileUpdateSuccess.php

```diff
@@ -0,0 +1,65 @@
+<?php decorate_with('layout_1col.php') ?>
+
+<?php slot('title') ?>
+  <h1 class="multiline">
+    <?php echo __('Update digital object titles') ?>
+    <span class="sub"><?php echo render_title(new sfIsadPlugin($resource)) ?> </span>
+  </h1>
+<?php end_slot() ?>
+
+<?php slot('content') ?>
+
+  <?php echo $digitalObjectTitleForm->renderGlobalErrors() ?>
+  <?php echo $digitalObjectTitleForm->renderFormTag(url_for(array($resource, 'module' => 'informationobject', 'action' => 'multiFileUpdate', 'ids' => $requestIds)), array('method' => 'post', 'id' => 'bulk-title-update-form')) ?>
```

Comment:
Why do we need to add the ids here? Could we handle them entirely in the controller? Could we use slugs instead?

Reply:
Updated to use slugs.